packages/frontend/src/components/ChooseConnectionSubstep/index.tsx

```diff
@@ -167,6 +171,20 @@ function ChooseConnectionSubstep(
     [step, onChange],
   )
 
+  const onRegisterWebhook = useCallback(async () => {
```

Comment:
Out of scope: This just struck my mind but with this, it's going to be very easy for people to point 2 forms to the same pipe with this feature.

I'm wondering if we should track the old connection ID, and _unregister_ the webhook with the old connection before registering with the new connection.

Reply:
Not sure how much more likely this would happen compared to the old flow. But the implication here is a higher number of "cannot decrypt form response" errors. Can monitor this for now.